It seems that recent versions of GNU patch no longer save rejects in
the case where a patch would create a file which already exists. I
don't know if this is on purpose or if this is a regression, but as
far as quilt goes we don't really care: let's just make the test
case more tolerant.

--- quilt-0.48.orig/test/create-delete.test
+++ quilt-0.48/test/create-delete.test
@@ -34,7 +34,7 @@ file creates and deletions.
 	> which already exists!  Assume -R? [n] 
 	> Apply anyway? [n] 
 	> Skipping patch.
-	>~ 1 out of 1 hunk ignored -- saving rejects to (file )?create.rej
+	>~ 1 out of 1 hunk ignored( -- saving rejects to (file )?create.rej)?
 	>~ The next patch would delete the file `?delete'?,
 	> which does not exist!  Assume -R? [n] 
 	> Apply anyway? [n] 
